app/routes/session.py

```diff
@@ -178,7 +179,21 @@ def get_sign_out():
 
 @session_blueprint.route("/signed-out", methods=["GET"])
 def get_signed_out():
-    return render_template(template="signed-out")
+    if not cookie_session:
+        return redirect(url_for("session.get_session_expired"))
+
+    business_survey_config = get_survey_config(theme=SurveyType.BUSINESS)
+    other_survey_config = get_survey_config(
+        theme=SurveyType.SOCIAL, base_url=ACCOUNT_SERVICE_BASE_URL_SOCIAL
+    )
+
+    business_logout_url = business_survey_config.account_service_todo_url
+    other_logout_url = other_survey_config.account_service_log_out_url
+    return render_template(
+        template="signed-out",
+        business_logout_url=business_logout_url,
+        other_logout_url=other_logout_url,
+    )
```

Comment:
You don't need to pass both URL, you just return the url for the current survey config. Since this page is only seen if they have a cooke session, therefore this can be:
```py
    survey_type = get_survey_type()
    survey_config = get_survey_config(theme=survey_type)
    logout_url = (
        survey_config.account_service_todo_url
        if survey_type is SurveyType.BUSINESS
        else survey_config.account_service_log_out_url
    )
    return render_template(
        template="signed-out",
        logout_url=logout_url,
    )
```